### PR TITLE
[WIP][15.0][MIG] purchase_stock_secondary_unit: Migration to V15

### PR DESCRIPTION
**Pull request by bizzappdev**

### CODE REVIEW

**Comment by ernestotejeda** [comment]
Functional review 👍🏼

**Review by ernestotejeda — COMMENTED** [review]
Functional review 👍🏼

**Review by ernestotejeda — CHANGES_REQUESTED** [review]
Change this commit message according to this convention https://github.com/OCA/maintainer-tools/wiki/Use-temporary-reference%28s%29-to-another-pull-request%28s%29

image

File: test-requirements.txt [hunk 1]
```diff
@@ -1 +1 @@
-odoo_test_helper
+odoo-addon-purchase_order_secondary_unit@ git+https://github.com/oca/purchase-workflow@refs/pull/1708/head#subdirectory=setup/purchase_order_secondary_unit
```
Comment (ernestotejeda) [inline test-requirements.txt]: Add here the PR for product_secondary_unit  compatibility with variants
```
odoo-addon-product_secondary_unit @ git+https://github.com/OCA/product-attribute.git@refs/pull/1246/head#subdirectory=setup/product_secondary_unit
```

**Review by ernestotejeda — CHANGES_REQUESTED** [review]

**Review by ernestotejeda — APPROVED** [review]

**Comment by ernestotejeda** [comment]
ping @sergio-teruel

File: purchase_stock_secondary_unit/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    "summary": "Get product quantities in a secondary unit",
+    "version": "15.0.1.0.0",
+    "development_status": "Beta",
+    "category": "stock",
+    "website": "https://github.com/OCA/purchase-workflow",
+    "author": "Jarsa, Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "application": False,
+    "installable": True,
+    "depends": ["purchase_order_secondary_unit", "stock_secondary_unit"],
```
Comment (sergio-teruel) [inline purchase_stock_secondary_unit/__manifest__.py:15]: Add purchase_stock module dependency

**Review by sergio-teruel — CHANGES_REQUESTED** [review]

**Review by carlosdauden — APPROVED** [review]
Code LGTM
Review dependences

**Comment by ernestotejeda** [comment]
Although adding purchase_stock module dependency is not required, I made a supersede PR to remove the commit that adds 'test-requirements.txt' and speed up the migration. https://github.com/OCA/purchase-workflow/pull/1867

**Comment by pedrobaeza** [comment]
Superseded by #1867

**Pull request closed without merging**